iothub/device/tests/Authentication/ClientAuthenticationWithTokenRefreshTests.cs

```diff
@@ -32,14 +33,29 @@ static ClientAuthenticationWithTokenRefreshTests()
         [TestMethod]
         public void ClientAuthenticationWithTokenRefresh_Ctor_WrongArguments_Fail()
         {
-            TestAssert.Throws<ArgumentNullException>(() => new TestImplementation(null));
```

Comment:
I'm able to get rid of this custom rolled `TestAssert` class by changing the asserts to use FluentAssertions.